src/lr.c

```diff
@@ -1,24 +1,39 @@
 #include "lr.h"
 #include <stdio.h>
 
+#define RETURN_IF_ERROR(x) do { \
```

Comment:
sure i can use lower case. I don't like `lr_assert` since it is similar in name to the `assert` library function, calls `abort` and terminates the program. 

what do you think of `return_if_error`

Reply:
`return_if_error` looks good

Reply:
I had to add some different macros and remove this one, but I've followed the lower snake case convention.